persistence/pre_persistence/proto/account.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package utility;
+option go_package = "/utility/test/";
+
+message Account {
+  bytes Address = 1;
+  string Amount = 2;
```

Comment:
Please add this comment: `# Amount is stored as a string in order to support BigInts at the application layer`.